### ISSUE TITLE
Having EXTENT in method definition parsed as an error

### ISSUE THREAD

**Issue by gustason** [BalticAmadeus/AblFormatter#234]
``` abl
class OpenEdge.DataAdmin.Rest.RestQueryFactory implements IRestQueryFactory: 
    method private char extent GetNamedQuery(pcname as char) :
        define variable cInfo as character extent no-undo.
        return cInfo.
    end.
end class.
```
![image](https://github.com/user-attachments/assets/74adee3e-3e34-4537-914c-ddeeb1372a9c)

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/96

**Comment by eglekaz** [comment]
Fixed with version 0.0.44 of tree-sitter-abl
